example/src/credo/BleProver.tsx

```diff
@@ -0,0 +1,50 @@
+import { useCentral } from '@animo-id/react-native-ble-didcomm'
+import type React from 'react'
+import { useEffect, useState } from 'react'
+import { Button, Text, View } from 'react-native'
+import { Spacer } from '../Spacer'
+import type { AppAgent } from './agent'
+import { bleShareProof } from './bleShareProof'
+import { closeBleTransports } from './utils/closeBleTransports'
+
+type BleProverProps = {
+  agent: AppAgent
+  serviceUuid: string
+}
+
+export const BleProver: React.FC<BleProverProps> = ({ agent, serviceUuid }) => {
+  const [hasSharedProof, setHasSharedProof] = useState(false)
+  const { central } = useCentral()
+
+  const onFailure = () => console.error('[CENTRAL]: failure')
+  const onConnected = () => console.log('[CENTRAL]: connected')
+  const onDisconnected = () => console.log('[CENTRAL]: disconnected')
```

Comment:
should we make the example a bit more in-depth? Would you usually do some reconnection stuff here?

Reply:
Really depends on the flow. I see them more as like UI update items. they are not too relevant.